### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by theVivek-kumar**
drawer-done

### CODE REVIEW

File: drawer/drawer.css [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    --GREEN: #01aa01;
+    --GREEN-50: #51f751d2;
+    --GREEN-20: #dff0d8;
+    --ORANGE: #ce8602;
+    --ORANGE-50: #f7bb4c;
+    --ORANGE-20: #fcf8e3;
+    --BLUE: #0000ff7e;
+    --BLUE-50: #5353f1;
+    --BLUE-80: #3d3df3;
+
```
Comment (pankaj-prog) [inline drawer/drawer.css:23]: You already have these variables declared in style which you are importing, You don't need to declare them again

File: drawer/drawer.css [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+.main-component {
+    display: flex;
+    flex-direction: column;
+    width: 255px;
+    margin: 8px 0px;
+
+}
+
+.component {
+    font-size: 18px;
```
Comment (pankaj-prog) [inline drawer/drawer.css:67]: use rems for font size

File: drawer/drawer.css [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+    margin: 2px 8px 8px;
+    padding: 0px 8px;
+    width: 239px;
+    height: 40px;
+    font-weight: 600;
+    color: rgb(22, 22, 22);
```
Comment (pankaj-prog) [inline drawer/drawer.css:73]: use color variables

**Review by pankaj-prog — APPROVED** [review]

**Comment by Shinde-Sandesh** [comment]
The code looks clean. Good to go

File: drawer/drawer.css [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+    width: 30%;
+    margin-right: auto;
+    margin-left: auto;
+    margin-top: 7rem;
+}
+
+.main-component {
+    display: flex;
+    flex-direction: column;
+    width: 255px;
```
Comment (Adarsh-Patel-Dev) [inline drawer/drawer.css:61]: use rem here

File: drawer/drawer.css [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+    margin: 8px 0px;
+
+}
+
+.component {
+    font-size: 18px;
+    margin: 2px 8px 8px;
+    padding: 0px 8px;
+    width: 239px;
```
Comment (Adarsh-Patel-Dev) [inline drawer/drawer.css:70]: same here

File: drawer/drawer.html [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Documentation</title>
+    <link rel="stylesheet" href="drawer.css">
+    <script src="https://kit.fontawesome.com/d5eef1e966.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+    <div class="navbar">
```
Comment (Adarsh-Patel-Dev) [inline drawer/drawer.html:14]: use nav tag here for navigation

**Review by Adarsh-Patel-Dev — APPROVED** [review]

**Pull request merged**